### PR TITLE
#160609567 show the reading time for an article's body

### PR DESCRIPTION
**Pull request by RonKbS**
### What does this PR do?
Show the time it will take to read an article

#### Description of Task to be completed?
Currently, readers can not see the estimated time it will take to read an article.
This feature displays the time it takes to read an article.

#### How should this be manually tested?
- Make migrations: python manage.py makemigrations articles.
- Migrate: python manage.py migrate.
- Run the server: python manage.py runserver.
- Open postman and post an article in the format below using a registered user ;
```
{
  "article": {
    	"title": "Testing django part 13",
    	"description": "It should work",
    	"body": "I am testing to confirm"
	}
}
```
-The readingTime should be shown as a field in the article returned in the response.

#### What are the relevant pivotal tracker stories?
[#160609567](https://www.pivotaltracker.com/story/show/160609567)

### CODE REVIEW

File: authors/apps/articles/count_words.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+
+def reading_time(body):
+    #split the words along new lines
```
Comment (MuhanguziDavid) [inline authors/apps/articles/count_words.py]: The method seems to be well defined. A comment may not be needed

File: authors/apps/articles/count_words.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+
+def reading_time(body):
+    new_line_words = body.split('\n')
+
+    total_words = 0
+    #Count the number of words in each string created in the list
```
Comment (MuhanguziDavid) [inline authors/apps/articles/count_words.py]: Please remove the comment.

File: authors/apps/articles/count_words.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    for group in new_line_words:
+        total_words += len(group.split())
+
+    #Assume a reading time of 150WPM
```
Comment (MuhanguziDavid) [inline authors/apps/articles/count_words.py]: Please update this comment

File: authors/apps/articles/count_words.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    reading_time = (total_words/275)
+    if reading_time < 1:
+        return 'Less than a minute'
+    elif reading_time < 2:
+        return 'About 1 minute'
+
+    #Return a value rounded up or down if below or above 5
```
Comment (MuhanguziDavid) [inline authors/apps/articles/count_words.py]: Add a space between the `#` and the beginning of the comment

File: authors/apps/articles/models.py [hunk 5]
```diff
@@ -23,6 +23,7 @@ class Article(models.Model):
     likes = models.PositiveIntegerField(default=0)
     dislikes = models.PositiveIntegerField(default=0)
     favourite_count = models.PositiveIntegerField(default=0)
+    readingTime = models.CharField(max_length=100, null=True)
```
Comment (MuhanguziDavid) [inline authors/apps/articles/models.py]: Follow the python convention for variable names (use snake case)

**Review by mendozabree — APPROVED** [review]
LGTM

**Comment by lym** [comment]
LGTM

**Review by samuelbwambale — APPROVED** [review]
LGTM

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Pull request merged**